### DISCUSSION TITLE
How to use tab index omnibar

### DISCUSSION THREAD

**Discussion by suimong** [gdh1995/vimium-c#1201] [Q&A] [answered]
The use case is that I often pin a few tabs in a given window, and it is easier to express "switch to the 2nd tab in the current window". In the omnibar, i can see the tab index information is displayed as "\#1, \#2(5)" etc., but currently i can't type "#2" in the omnibar and get a match for the second tab. Is such feature currently supported?

**Comment by gdh1995** [comment, marked as answer]
Um, you may use `Alt+Number` to select the `N-th` suggestion on Vomnibar.

Or you may just press `3g0` (on a web page, with the default key mapping) to trigger the `firstTab` with `number-prefix = 3`, and then Vimium C will go to the 3rd tab on a current window.

**Reply by suimong** [reply]
`3g0` is exactly what I need. Thanks for answering!
